libflux/src/semantic/mod.rs

```diff
@@ -19,3 +19,23 @@ mod parser;
 
 #[cfg(test)]
 mod tests;
+
+use crate::ast;
+use crate::parser;
+use crate::semantic::analyze::Result;
+use crate::semantic::fresh::Fresher;
+
+pub fn analyze_string(source: &str) -> Result<nodes::Package> {
```

Comment:
Should this be `analyze_source`?

Reply:
I wanted to keep consistency with `parse_string` 🤔